managed-databases/postgresql-and-mysql/reference-content/encryption-at-rest-performance-benchmark.mdx

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  title: Encryption at rest performance benchmark on Database Instances
+  description: This page shows benchmarks of different Database Instances' performance when encryption at rest is enabled.
+content:
+  h1: Encryption at rest performance benchmark on Database Instances
+  paragraph: This page shows benchmarks of different Database Instances' performance when encryption at rest is enabled.
+tags: managed-database postgresql mysql encryption at rest
+dates:
+  validation: 2024-12-19
+  posted: 2024-12-19
+categories:
+  - managed-databases
+  - postgresql-and-mysql
+---
+
+The Scaleway Managed Databases team ran tests on DB-POP2-4C-16G Instances in a Private Network using `pgbench`. Several workloads (including read, oltp and large loads) were simulated with several scale factors (from 1M rows to 200M rows) and concurrent database connections set to 10. They measured the impact of encryption at rest on transactions, per second and latency.
```

Comment:
We can add here that these are maximal numbers and that no one during the Public Beta complained about any Performances issues. If they perform their own benchmarks, we are eager to see their figures and they should not hesitate to share them on our slack community channel